src/templates/match-suggested-email.html

```diff
@@ -3,29 +3,23 @@
 <p>Dear {%YOUR_NAME%},</p>
 <p>You’ve been matched with a compatible donor through the EA donation swap site.</p>
 <p></p>
-<p>The potential match:</p>
-<p>
-	Charity they wish to support: {%THEIR_CHARITY%}<br>
-	Amount: {%THEIR_AMOUNT%} {%THEIR_CURRENCY%}<br>
-	This is around {%THEIR_AMOUNT_CONVERTED%} in {%YOUR_CURRENCY%}.<br>
-	This amount may change a little due to currency fluctuations.</p>
-
+<p>If both of you accept the match, you will be donating
+{%YOUR_ACTUAL_AMOUNT%} {%YOUR_CURRENCY%} to {%THEIR_CHARITY%},
```

Comment:
I'll merge it now, but keep in mind that `{%YOUR_ACTUAL_AMOUNT%}` does not yet take gift aid into account.